### PR TITLE
add import snapshot file lines num check

### PR DESCRIPTION
**Pull request by nkysg**
## Pull request type

Please check the type of change your PR introduces:

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: https://github.com/starcoinorg/starcoin/issues/3539

## What is the new behavior?

-
-
-

## Other information
need modify scripts/sync_block.py。It's describe in https://github.com/starcoinorg/starcoin/issues/3539  @yourmoonlight

### CODE REVIEW

**Review by templexxx — APPROVED** [review]

**Comment by nkysg** [comment]
@yourmoonlight  please help me modify sync_block.py, it describe in https://github.com/starcoinorg/starcoin/issues/3539

File: cmd/db-exporter/src/main.rs [hunk 1]
```diff
@@ -1387,6 +1387,19 @@ pub fn apply_snapshot(
         }
         file_list.push((column, nums, verify_hash));
     }
+
+    for (file_name, nums, _) in file_list.iter() {
+        let file = BufReader::new(File::open(input_path.join(file_name))?);
+        let mut cnt = 0;
+        for _ in file.lines() {
+            cnt += 1;
+        }
```
Comment (jolestar) [inline cmd/db-exporter/src/main.rs:1396]: let cnt = file.lines().count()?

Comment (nkysg) [reply cmd/db-exporter/src/main.rs:1396]: fixed

**Review by jolestar — APPROVED** [review]

**Comment by yourmoonlight** [comment]
> @yourmoonlight please help me modify sync_block.py, it describe in #3539

Ok, I will take a look.

**Pull request merged**